packages/billing/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@open-condo/billing",
+  "description": "A set of billing and acquiring tools that shared between apps",
+  "keywords": [
+    "condo",
+    "open-condo-software",
+    "packages",
+    "billing",
+    "tools"
+  ],
+  "scripts": {
+    "test": "jest",
+    "build": "tsup"
+  },
+  "version": "0.0.0-development",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/open-condo-software/condo.git",
+    "directory": "packages/billing"
+  },
+  "bugs": {
+    "url": "https://github.com/open-condo-software/condo/issues"
+  },
+  "author": {
+    "name": "Alexander Naydenov",
+    "email": "nas@doma.ai",
+    "url": "https://github.com/AleX83Xpert"
+  },
+  "license": "MIT",
+  "files": [
+    "dist"
+  ],
+  "typesVersions": {
+    "*": {
+      "tools/billingCentrifuge": [
+        "dist/tools/billingCentrifuge.d.ts"
+      ]
+    }
+  },
+  "types": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "exports": {
+    "./tools/billingCentrifuge": {
+      "types": "./dist/tools/billingCentrifuge.d.ts",
+      "require": "./dist/tools/billingCentrifuge.js",
+      "import": "./dist/tools/billingCentrifuge.mjs"
+    }
+  },
+  "dependencies": {
+    "big.js": "^6.2.2",
+    "lodash": "^4.17.21"
+  },
+  "devDependencies": {
+    "@faker-js/faker": "^9.6.0",
```

Comment:
It's better to keep major versions in sync with others, to you dont need to switch context between breaking changes while writing the code

Reply:
7.6

Reply:
Downgraded 😢